ApollosProject/apollos-prototype
# Adds Redis Cache and stores isLiked / likeCount in RedisCache.

## DESCRIPTION

### What does this PR do, or why is it needed?

- Provides a connector to Redis for storing data to improve performance. 
- Implements the connector for the following module, caches the `likeCount` and `isLiked` for content items. Should improve performance dramatically on the home feed and discover feed.

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

- Open the app to the home feed with redis running.
- Note the number of calls to the `following` endpoint
- Reload the page, you will see that there are no calls to the `following` endpoint in your server console the second time. 
- Like an item, and make sure the like persists. 
- Unlike an item, and make sure the unlike persists. 

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
